tricore: cs4 & cs5 building error

```
gcc -Ilibrz/analysis/librz_analysis.so.0.7.0.p -I. -I.. -Ilibrz -I../librz -Ilibrz/include -I../librz/include -I../librz/analysis/arch/gb -Ilibrz/util/sdb/src -I../librz/util/sdb/src -I../librz/asm/arch/include -I../librz/asm/arch -I../librz/asm/arch/h8300 -I../librz/asm/arch/hexagon -I../librz/asm/arch/msp430 -I../librz/asm/arch/rsp -I../librz/asm/arch/mcore -I../librz/asm/arch/v850 -I../librz/asm/arch/propeller -I../librz/asm/arch/ebc -I../librz/asm/arch/cr16 -I../librz/asm/arch/8051 -I../librz/asm/arch/v810 -I../librz/asm/arch/or1k -I../librz/asm/arch/tricore -I../librz/bin/format -I../librz/type/parser -I../subprojects/capstone-5.0.1/include -I../subprojects/capstone-5.0.1/include/capstone -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -O3 -Wimplicit-fallthrough=3 -DRZ_PLUGIN_INCORE=1 -D_GNU_SOURCE --std=gnu99 -Werror=sizeof-pointer-memaccess -fvisibility=hidden -Wno-cpp -fPIC -MD -MQ librz/analysis/librz_analysis.so.0.7.0.p/arch_tricore_tricore_il.c.o -MF librz/analysis/librz_analysis.so.0.7.0.p/arch_tricore_tricore_il.c.o.d -o librz/analysis/librz_analysis.so.0.7.0.p/arch_tricore_tricore_il.c.o -c ../librz/analysis/arch/tricore/tricore_il.c
In file included from ../librz/asm/arch/../../asm/arch/tricore/tricore.inc:4,
                 from ../librz/analysis/arch/tricore/tricore_il.c:5:
../subprojects/capstone-5.0.1/include/capstone/tricore.h:23:30: error: ‘CS_OP_INVALID’ undeclared here (not in a function)
   23 |         TRICORE_OP_INVALID = CS_OP_INVALID, ///< CS_OP_INVALID (Uninitialized).
      |                              ^~~~~~~~~~~~~
../subprojects/capstone-5.0.1/include/capstone/tricore.h:24:26: error: ‘CS_OP_REG’ undeclared here (not in a function)
   24 |         TRICORE_OP_REG = CS_OP_REG,         ///< CS_OP_REG (Register operand).
      |                          ^~~~~~~~~
../subprojects/capstone-5.0.1/include/capstone/tricore.h:25:26: error: ‘CS_OP_IMM’ undeclared here (not in a function)
   25 |         TRICORE_OP_IMM = CS_OP_IMM,         ///< CS_OP_IMM (Immediate operand).
      |                          ^~~~~~~~~
../subprojects/capstone-5.0.1/include/capstone/tricore.h:26:26: error: ‘CS_OP_MEM’ undeclared here (not in a function)
   26 |         TRICORE_OP_MEM = CS_OP_MEM,         ///< CS_OP_MEM (Memory operand).
      |                          ^~~~~~~~~
```
```
../librz/asm/arch/8051 -I../librz/asm/arch/v810 -I../librz/asm/arch/or1k -I../librz/asm/arch/tricore -I../librz/bin/format -I../librz/type/parser -I/usr/include/capstone -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -O3 -Wimplicit-fallthrough=3 -DRZ_PLUGIN_INCORE=1 -DUSE_SYS_CAPSTONE -D_GNU_SOURCE --std=gnu99 -Werror=sizeof-pointer-memaccess -fvisibility=hidden -Wno-cpp -fPIC -MD -MQ librz/analysis/librz_analysis.so.0.7.0.p/arch_tricore_tricore_il.c.o -MF librz/analysis/librz_analysis.so.0.7.0.p/arch_tricore_tricore_il.c.o.d -o librz/analysis/librz_analysis.so.0.7.0.p/arch_tricore_tricore_il.c.o -c ../librz/analysis/arch/tricore/tricore_il.c
In file included from ../librz/analysis/arch/tricore/tricore_il.c:5:
../librz/asm/arch/../../asm/arch/tricore/tricore.inc:4:10: fatal error: capstone/tricore.h: No such file or directory
    4 | #include <capstone/tricore.h>
      |          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/librz/analysis/meson.build
+++ b/librz/analysis/meson.build
@@ -170,7 +170,6 @@ rz_analysis_sources = [
   'arch/x86/x86_il.c',
   'arch/8051/8051_il.c',
   'arch/8051/8051_parse.c',
-  'arch/tricore/tricore_il.c',
   '../asm/arch/amd29k/amd29k.c',
   '../asm/arch/avr/disassembler.c',
   '../asm/arch/cil/cil_dis.c',
@@ -243,6 +242,7 @@ rz_analysis_sources = [
 if capstone_dep.version() == 'next' or capstone_dep.version().split('.')[0].to_int() > 4
   rz_analysis_sources += 'p/analysis_riscv_cs.c'
   rz_analysis_sources += 'p/analysis_tricore_cs.c'
+  rz_analysis_sources +='arch/tricore/tricore_il.c'
 endif
 
 if get_option('use_gpl')

--- a/librz/asm/arch/tricore/tricore.inc
+++ b/librz/asm/arch/tricore/tricore.inc
@@ -1,6 +1,7 @@
 // SPDX-FileCopyrightText: 2023 billow <billow.fun@gmail.com>
 // SPDX-License-Identifier: LGPL-3.0-only
 
+#include <capstone/capstone.h>
 #include <capstone/tricore.h>
 #include "tricore.h"
 

